custom_components/rfplayer/manifest.json

```
@@ -1,12 +1,17 @@
 {
   "domain": "rfplayer",
-  "name": "GCE RFPlayer",
-  "codeowners": ["@Aohzan"],
+  "name": "RfPlayer",
+  "codeowners": ["@racletteparty"],
   "config_flow": true,
-  "dependencies": [],
-  "documentation": "https://github.com/gce-electronics/HA_RFPlayer",
-  "iot_class": "assumed_state",
-  "issue_tracker": "https://github.com/gce-electronics/HA_RFPlayer/issues",
-  "requirements": ["pyserial==3.5", "pyserial-asyncio==0.6"],
-  "version": "0.7.0"
+  "documentation": "https://github.com/racletteparty/HacsRfPlayer",
+  "integration_type": "hub",
+  "iot_class": "local_push",
+  "issue_tracker": "https://github.com/racletteparty/HacsRfPlayer/issues",
+  "loggers": ["rfplayer"],
+  "requirements": [
+    "pyserial==3.5",
+    "pyserial-asyncio==0.6",
+    "jsonpath-ng==1.6.1"
+  ],
+  "version": "0.0.0"
```

```suggestion
  "version": "1.0.0"
```
je propose de passer en 1.0.0

---

cette version est juste un placeholder. ca n'est pas ca qui est utilisé pour définir la version. dans le workflow de release, il y a une commande jq qui remplace la version dans le manifest HA par le tag de version de release.
Dans l'idée, il faudrait passer en trunk-based et faire une release à chaque PR terminée en utilisant semantic release pour calculer automatiquement le version. Mais j'ai pas eu le temps de mettre en place.

---

comme tu veux suivant le temps que tu peux mettre